resources/lang/zh/beatmaps.php

```diff
@@ -34,10 +34,11 @@
     'discussions' => [
         'allow_kudosu' => '给予 kudosu',
         'delete' => '删除',
-        'deleted' => '被 :editor 于 :delete_time 删除',
+        'deleted' => '被 :editor 于 :delete_time 删除。',
         'deny_kudosu' => '收回 kudosu',
         'edit' => '编辑',
-        'edited' => '最后由 :editor 编辑于 :update_time',
+        'edited' => '最后由 :editor 编辑于 :update_time 。',
```

Comment:
extra whitespaces intentional?

Reply:
ehhh, i check some context, remove the full stop(`。`) would be better.

will fix it later today